website/static/js/share/stats.js

```diff
@@ -79,20 +87,107 @@ function timeGraph (data) {
             show: false
         },
         tooltip: {
-            grouped: false,
-            format: {
-              name: function (name, ratio, id, index) {
-                  if (name === 'pubmed') {
-                      name = 'pubmed central';
-                  }
-                  return name; 
-              }
-            }
+            grouped: false
         }
     });
 }
 
-Stats.view = function(ctrl) {
+/* Creates an Elasticsearch aggregation by source */
+Stats.sourcesAgg = function () {
```

Comment:
This function always returns the same thing; would it make more sense to make this a constant?